models/repos.js

```diff
@@ -22,6 +22,7 @@ module.exports = createModel
 
 function createModel () {
   let manager = null
+  let dbPaused = null
```

Comment:
seems easier to use a separate db for this, than integrating into multidrive and multidat